acceptance/tests/ingress-gateway/ingress_gateway_namespaces_test.go

```diff
@@ -60,25 +64,6 @@ func TestIngressGatewaySingleNamespace(t *testing.T) {
 
 			consulClient, _ := consulCluster.SetupConsulClient(t, c.secure)
 
-			// Create the destination namespace in the non-secure case.
-			// In the secure installation, this namespace is created by the server-acl-init job.
-			if !c.secure {
-				logger.Logf(t, "creating the %s namespace in Consul", testNamespace)
-				_, _, err := consulClient.Namespaces().Create(&api.Namespace{
-					Name: testNamespace,
-				}, nil)
-				require.NoError(t, err)
-			}
```

Comment:
Oh nice! because endpoints controller will create the ns now, right?

Reply:
exactly!!